v3/src/components/slider/slider-registration.ts

```diff
@@ -15,14 +13,10 @@ import { SliderModel } from "./slider-model"
 import { SliderTitleBar } from "./slider-title-bar"
 import { AnimationDirections, AnimationModes, kDefaultAnimationDirection, kDefaultAnimationMode } from "./slider-types"
 import SliderIcon from '../../assets/icons/icon-slider.svg'
+import { getGlobalValueManager } from "../../models/tiles/tile-environment"
```

Comment:
Move import ⬆️ with the other `../../models/tiles/...` imports.

Reply:
Fixed in the last commit.